rest/route/github.go

```diff
@@ -239,17 +240,17 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 					return gimlet.MakeJSONErrorResponder(err)
 				}
 			}
-			if triggersRetry(*event.Action, *event.Comment.Body) {
+			if triggersPatch(*event.Action, *event.Comment.Body) {
 				grip.Info(message.Fields{
 					"source":    "github hook",
 					"msg_id":    gh.msgID,
 					"event":     gh.eventType,
 					"repo":      *event.Repo.FullName,
 					"pr_number": *event.Issue.Number,
 					"user":      *event.Sender.Login,
-					"message":   "retry triggered",
+					"message":   "patch triggered",
 				})
-				if err := gh.retryPRPatch(ctx, event.Repo.Owner.GetLogin(), event.Repo.GetName(), event.Issue.GetNumber()); err != nil {
+				if err := gh.createPRPatch(ctx, event.Repo.Owner.GetLogin(), event.Repo.GetName(), patch.CalledManually, event.Issue.GetNumber()); err != nil {
```

Comment:
"evergreen retry" should still be patch.CalledAutomatically actually in terms of workflow; I know it is user triggered but its part of the automated workflow as opposed to part of the "only if manual is checked will this work" workflow.

Reply:
done (changed what triggersPatch returns) not sure if that's what we want to do but if we add more types later it would be easiest this way but it makes sense because we should be able to retry even if only the auto is enabled